docs/api.md

```diff
@@ -2,6 +2,14 @@
 
 Please have a look at the documentation and examples of the interactive Swagger UI: [https://oqt.ohsome.org/api/docs](https://oqt.ohsome.org/api/docs)
 
+## A note on the difference of `GET` and `POST` requests to the `indicator` or `report` endpoints
+
+To the endpoints `indicator` and `report` a request can be made using either the `GET` or `POST` method.
```

Comment:
Do we use `/indicator` or `indicator` for endpoints? I think we used the first variant until now, but I'm okay with both.

Reply:
I changed to the `/indicator` notation. Could you have another look?